### ISSUE TITLE
Customize item url?  Linking between multiple md output files.

### ISSUE THREAD

**Issue by justinaray** [jsdoc2md/jsdoc-to-markdown#141]
I am trying to create a dist directory as follows:

```
dist/docs
  index.md
  MyBaseClass.md
  MyExtendingClass.md
```

I have been able to use `getTemplateData` to extract just the classes from my multi-file, es6 modules baseline and write them into separate md files.  However, the links to the other classes (annotated with `@type`) are still rendered as internal anchor links.  I am assuming this is because the types were part of the `files` input when building the template data.

I had a success changing the types to `{external:MyBaseClass}` and using `@see ./MyBaseClass.md` tag, but this is error prone and feels like I'm hacking the true jsdoc relationship for a rendering concern.

Is there anyway to customize the url of objects by type?

Thanks in advance!  Love the lib!

**Comment by 75lb** [comment]
hi.. first of all, I would forget about jsdoc2md temporarily and render your docs using jsdoc checking everything looks good there.. 

once you're happy your jsdoc is correct, yes you can customise links and any part of the jsdoc2md output.. take a look at the wiki, maybe start [here](https://github.com/jsdoc2md/jsdoc-to-markdown/wiki/Create-a-custom-partial).

**Comment by justinaray** [comment]
Thanks for the reply @75lb.  I was originally customizing the partials, but kept finding places I needed to override to correctly link within my output structure.  I was worried it was error prone and would make getting bug fixes more difficult.

Eventually, I settled on overridding the link helper to handle my my special cases or pass through.  Seems to be working pretty well!

Thanks again.  Really impressed by the extensibility and completeness of this lib!
